engine/requirements.in

```diff
@@ -29,20 +29,20 @@ drf-spectacular==0.26.5
 emoji==2.4.0
 # If the version of grpcio is changed
 # upload a new arm64 wheel instead of /engine/grpcio-1.57.0-cp311-cp311-linux_aarch64.whl
-grpcio==1.57.0
+grpcio==1.64.1
 fcm-django @ https://github.com/grafana/fcm-django/archive/refs/tags/v1.0.12r1.tar.gz#sha256=7ec7cd9d353fc9edf19a4acd4fa14090a31d83d02ac986c5e5e081dea29f564f
 hiredis==2.2.3
 humanize==0.5.1
 icalendar==5.0.10
-lxml==4.9.2
+lxml==5.2.2
 markdown2==2.4.10
-opentelemetry-sdk==1.23.0
-opentelemetry-api==1.23.0
-opentelemetry-exporter-otlp-proto-grpc==1.15.0
-opentelemetry-instrumentation-logging==0.44b0
-opentelemetry-instrumentation-wsgi==0.44b0
-opentelemetry-instrumentation-requests==0.44b0
-opentelemetry-instrumentation-django==0.44b0
+opentelemetry-sdk==1.25.0
+opentelemetry-api==1.25.0
+opentelemetry-exporter-otlp-proto-grpc==1.25.0
+opentelemetry-instrumentation-logging==0.46b0
+opentelemetry-instrumentation-wsgi==0.46b0
+opentelemetry-instrumentation-requests==0.46b0
+opentelemetry-instrumentation-django==0.46b0
```

Comment:
the `opentelemetry` packages weren't working on 3.12 so bumped them to a newer version

https://github.com/grafana/oncall/actions/runs/9451161174/job/26031450875?pr=4495#step:5:24